fabtests/functional/multi_ep.c

```diff
@@ -207,19 +228,25 @@ static int do_transfers(void)
 
 static int setup_client_ep(int idx)
 {
-	int ret;
+	int ret, av_bind_idx = idx, cq_bind_idx = idx;
+
+	if (share_cq)
+		cq_bind_idx = 0;
+
+	if (share_av)
+		av_bind_idx = 0;
 
 	ret = fi_endpoint(domain, fi, &eps[idx], NULL);
 	if (ret) {
 		FT_PRINTERR("fi_endpoint", ret);
 		return ret;
 	}
 
-	ret = ft_alloc_ep_res(fi, &txcqs[idx], &rxcqs[idx], NULL, NULL, NULL, &av);
+	ret = ft_alloc_ep_res(fi, &txcqs[idx], &rxcqs[idx], NULL, NULL, NULL, &avs[idx]);
 	if (ret)
 		return ret;
 
-	ret = ft_enable_ep(eps[idx], eq, av, txcqs[idx], rxcqs[idx],
+	ret = ft_enable_ep(eps[idx], eq, avs[av_bind_idx], txcqs[cq_bind_idx], rxcqs[idx],
```

Comment:
should use `cq_bind_idx` for `rxcqs` as well.

Reply:
Oops... Fixed!